### DISCUSSION TITLE
I don't understand how to play these games

### DISCUSSION THREAD

**Discussion by JoaoFelipe3** [haskell-game/tiny-games-hs#124] [Q&A] [answered]
In the how to play section, it tells me "in this repository". But how do I access the repo?

**Comment by simonmichael** [comment, marked as answer]
Thanks, I have updated https://github.com/haskell-game/tiny-games-hs#how-to-play with the answer (`git clone https://github.com/haskell-game/tiny-games-hs`). Ok now ?

**Comment by JoaoFelipe3** [comment]
Ok. Close this discussion now.
